build/token.js

```diff
@@ -75,6 +75,12 @@ module.exports = getToken = function(arg) {
       if (!isValid) {
```

Comment:
Can you add the `header: true` bit to the gulpfile so this file's diff gets dropped please